admin-dev/themes/new-theme/scss/theme.scss

```diff
@@ -182,3 +182,78 @@ table.table {
     }
   }
 }
+
+//** Background color of the whole progress component
```

Comment:
It's good that you wrapped everything in a separated part

Even better would be to put this in a dedicated scss file and include it at the top of this global theme file, or maybe you should use this existing file https://github.com/PrestaShop/PrestaShop/blob/eb7a94b9790d73f16b7830af06cbaf4744ff66d8/admin-dev/themes/new-theme/scss/components/_modal.scss